src/androidTest/java/de/dennisguse/opentracks/io/file/exporter/ExportAsyncTaskTest.java

```diff
@@ -0,0 +1,353 @@
+package de.dennisguse.opentracks.io.file.exporter;
+
+import android.content.Context;
+import android.util.Pair;
+
+import androidx.documentfile.provider.DocumentFile;
+import androidx.test.core.app.ApplicationProvider;
+
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+
+import de.dennisguse.opentracks.content.data.TestDataUtil;
+import de.dennisguse.opentracks.content.data.Track;
+import de.dennisguse.opentracks.content.data.TrackPoint;
+import de.dennisguse.opentracks.content.provider.ContentProviderUtils;
+import de.dennisguse.opentracks.io.file.TrackFileFormat;
+import de.dennisguse.opentracks.stats.TrackStatistics;
+import de.dennisguse.opentracks.util.FileUtils;
+import de.dennisguse.opentracks.util.TrackNameUtils;
+
+public class ExportAsyncTaskTest {
```

Comment:
RunWith annotation is missing.
Is it not necessary?

Add a comment that this track deletes all data.
See #143 (also add a comment there).

Reply:
> RunWith annotation is missing.
> Is it not necessary?

I sincerely don't know, I don't notice any difference but I annotated it. I'm learning Java Testing on the way (I'm mainly Python/Django developer) :sweat_smile:


> Add a comment that this track deletes all data.
> See #143 (also add a comment there).

Why does testing use the Application Context? Shouldn't testing use Mock Context or Instrumentation Context? Maybe is this the issue? I'm really newbie in these issues so sorry if I'm saying a stupid thing :relaxed:

Anyway, commented done.